introductory/src/listComponent.jsx

```diff
@@ -0,0 +1,8 @@
+export default function List(props){
```

Comment:
Please consider code consistency. Could you stick to one style of export? You can either export the function by default inline or export it at the end
```suggestion
function List(props){
  // do something
}
export default List
```